packages/zeebe-element-templates-json-schema/src/defs/element-type.json

```diff
@@ -12,13 +12,6 @@
     },
     {
       "if": {
-        "properties": {
-          "eventDefinition": {
-            "enum": [
-              "bpmn:MessageEventDefinition"
-            ]
-          }
-        },
```

Comment:
This is redundant because. If event definition is set, the value has to be one of the events regardless of the event definition value.